frontend/src/libraries/tester.tsx

```diff
@@ -16,17 +14,15 @@ const networkContext = {
   networks: [],
 };
 
-export const renderI18nElement = (
+export const renderElement = (
```

Comment:
It seems like it's just a base providers wrapper.
We should probably refactor "base providers" out as a separate component to reuse it in real code and unit testing. Or we should not :)